src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] result = countData(dataGetter(fromFileName));
+        File file = new File(toFileName);
+        if (file.exists()) {
+            file.delete();
+        }
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file, true))) {
+            for (String each : result) {
+                bufferedWriter.write(each);
+                bufferedWriter.write(System.lineSeparator());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data");
+        }
```

Comment:
move this logic to a separate private method which will be responsible for writing